src/app/coursegrab/dao/users_dao.py

```diff
@@ -10,12 +10,14 @@ def get_user_by_email(email):
     return User.query.filter(User.email == email).first()
 
 
+def get_tracked_courses(user_id):
+    return get_user_by_id(user_id).courses
```

Comment:
what happens when this id doesn't exist? is that a case that might happen?

Reply:
this function is only called on endpoints with the `@authorize_user` decorator, so I don't think that case is possibleo